_projects/2024/100543010/100543010.Rmd

```diff
@@ -0,0 +1,897 @@
+             color = "violet", fill = "black") +
+  annotate("text", x = 12200000 -900000, y = 2350000 + 180000, 
+           label = "8", color = "white") +
+  annotate("richtext", x= 12200000, y = 2350000,
+           label = "Taiwan<br><b>$130B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA) +
+  
+  geom_point(x = 10750000 , y = 1500000 + 700000, 
+             size = 6.5, stroke = 1, shape = 21,
+             color = "violet", fill = "black") +
+  annotate("text", x = 10750000 , y = 1500000 + 700000, 
+           label = "9", color = "white") +
+  annotate("richtext", x= 10750000, y = 1500000,
+           label = "Vietnam<br><b>$129B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA) +
+  
+  geom_point(x = 7450000 -390000, y = 2400000 + 400000, 
+             size = 6.5, stroke = 1, shape = 21,
+             color = "violet", fill = "black") +
+  annotate("text", x = 7450000 -390000, y = 2400000 + 400000, 
+           label = "10", color = "white") +
+  annotate("richtext", x= 7450000, y = 2200000,
+           label = "India<br><b>$127B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA,
+           color = "white") +
+  
+  annotate("richtext", x= 7300000, y = 6500000,
+           label = "Russia<br><b>$5.5B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA,
+           color = "white") +
+  
+  
+  annotate("richtext", x= -4000000, y = 4000000,
+           label = "EU-27<br><b>$960B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA) +
+  
+  annotate("richtext", x= -4800000, y = -850000,
+           label = "Brazil<br><b>$86B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA,
+           color = "white") +
+  
+  annotate("richtext", x= 3250000, y = -4000000,
+           label = "South Africa<br><b>$21B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA) +
+  
+  annotate("richtext", x= 12500000, y = -2500000,
+           label = "Australia<br><b>$50B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA,
+           color = "white") +
+  
+  annotate("richtext", x= -900000, y = 5000000,
+           label = "France<br><b>$104B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA) +
+  
+  annotate("richtext", x= -1400000, y = 4250000,
+           label = "Spain<br><b>$49B</b>",
+           size = font_size,
+           lineheight = 0.8,
+           fill = NA,
+           label.color = NA) +
+  
+  annotate("richtext", x= 1500000, y = 3500000,
```

Comment:
As a suggestion (because I see a lot of annotations again that may be repeated, but I'm not sure, I didn't check so closely), if you find yourself repeating some code, you could save that into a variable and then apply it later. E.g.

```r
annotations <-
  annotate(...) +
  annotate(...) +
  ...
```

then

```r
some_plot + annotations
```

Works for annotations, theme, and other layers of course.